docs/backend/guides/how-to/set-up-a-feature-flag.md

```diff
@@ -9,13 +9,18 @@ There is a feature flag feature for both frontend and backend.  Here's how to se
 
 ### Overview
 
-The frontend feature flag can be turned on or off through the url.  By appending a querystring `?flag:flagName=false`, the feature will be turned off and appending `?flag:flagName=true` will turn on the feature.  Generally, while working on a feature that is not ready for production, by default we set it to off in production and turn it on in dev, staging, and experimental environments.
```

Comment:
Is the difference between these just making the paragraph multiline? I want to make sure i'm not misunderstanding something here.

Reply:
https://github.com/transcom/mymove-docs/pull/87/files#r783346921